### ISSUE TITLE
  databricks bundle init mlops-stacks // Error: variable "input_cloud" not defined

### ISSUE THREAD

**Issue by rjarpa** [databricks/mlops-stacks#134]
Hello! 
Databricks CLI v0.211.0.
Python 3.10.0

When I trigger the initialization of the project with `databricks bundle init mlops-stacks`

I received the following message:

```
Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md.
Error: variable "input_cloud" not defined
```

**Comment by arpitjasa-db** [comment]
Hi @rjarpa! I'm having a bit of trouble reproducing this. Would you mind confirming using `which databricks` and `databricks -v` that you are using the Databricks CLI v0.211.0. as mentioned above?

**Comment by rjarpa** [comment]
Hi @arpitjasa-db! 

I did the test in two different Windows 10 machines using bash terminal version `GNU bash, version 4.4.23(2)-release (x86_64-pc-msys)` inside of VS Code,  result when I run commands:
`which databricks` -> /c/Windows/databricks
`databricks -v` -> Databricks CLI v0.211.0

In both machine the same problem. Looks about the problem happened only in the bash terminal of VS Code.

I opened a Command Prompt in Windows and execute `databricks bundle init mlops-stacks` works fine showing `Project Name [my-mlops-project]:` prompt.

I tried to do the same in VS Code in MacOS and no problem the only different is bash version in MacOs is `GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin23)`

In that case I can move forward if I dont use bash `GNU bash, version 4.4.23(2)-release (x86_64-pc-msys)` inside of VS Code.

**Comment by arpitjasa-db** [comment]
@rjarpa thank you for all of those details. Likely this makes me think there's some dependency issue or CLI issue rather than an MLOps Stacks issue. I'll pass this along to our CLI team to dig into deeper, but glad to hear that there is a workaround! I'll update this thread when I have more details.

**Comment by rjarpa** [comment]
Yeah looks something with CLI and not MLops Stacks. Thanks for you fast response @arpitjasa-db !

**Comment by shreyas-goenka** [comment]
Thank you for reporting this issue @rjarpa. I tried out `databricks bundle init mlops-stacks` using `GNU bash, version 5.2.15(2)-release (x86_64-pc-msys)` on windows. It works for me on both vscode and the normal terminal. 

Some context around the error: It seems like the CLI mistakenly thinks that the current terminal session is not interactive. I suspect updating to the latest msys2 distribution should solve this issue for you. Please try it out and late us know if it works for you :)

**Comment by rjarpa** [comment]
Hello @shreyas-goenka,

Thanks for your answer, extra info: the bash console I'm using is git bash that is distributed with git installer. I run `git update` to update git bash terminal to `GNU bash, version 5.2.21(1)-release (x86_64-pc-msys)`

Still issue arise anyway even outside of  VS Code the results between. below the differents screen between `git bash` and `command prompt`

![databricks_bundle_git_bash](https://github.com/databricks/mlops-stacks/assets/10712346/e63a6ac7-e7ea-479e-8712-e4f67b2a4ff9)

![databricks_bundle_cmd](https://github.com/databricks/mlops-stacks/assets/10712346/4f9394ec-a5b0-4796-bcc7-675265f0a209)



I'm moving forward by now with Windows Command Prompt, but could useful for you this info :) .
